lib/metamodel.cto

```diff
@@ -224,6 +224,19 @@ concept ImportTypes extends Import {
   o String[] types
 }
 
+concept AliasType{
```

Comment:
I think you'd need to generate the corresponding change in the `metamodel.json` too.